torch/nn/modules/module.py

```diff
@@ -418,7 +418,15 @@ def forward(self, x):
     _backward_hooks: Dict[int, Callable]
     _is_full_backward_hook: Optional[bool]
     _forward_hooks: Dict[int, Callable]
+    # Marks whether the corresponding _forward_hooks accept kwargs or not. All
```

Comment:
nit: dangling "All"?
```suggestion
    # Marks whether the corresponding _forward_hooks accept kwargs or not.
```
(same with below for `_forward_pre_hooks_with_kwargs`)